modules/oxide/src/less/theme/globals/global.less

```diff
@@ -61,6 +61,10 @@
   position: absolute;
   right: 0;
   top: 0;
+
+  @media (forced-colors: active) {
+    border: 2px solid highlight;
```

Comment:
I think `@keyboard-focus-outline-width` should be used instead of the hard coded value